tap_mssql/sync_strategies/log_based.py

```diff
@@ -347,7 +361,7 @@ def sync_table(mssql_conn, config, catalog_entry, state, columns, stream_version
                                 FROM cdc.fn_cdc_get_all_changes_{}(@from_lsn, @to_lsn, 'all')
                                 ORDER BY __$start_lsn, __$seqval, __$operation
                                 ;""".format(
-                    py_bin_to_mssql(state_last_lsn),
+                    from_lsn_expression,

```

Comment:
Use the new `from_lsn_expression` which is calculated earlier - and is not always incremented as it formerly was.